manifests/base/jupyter-trustyai-notebook-imagestream.yaml

```diff
@@ -13,13 +13,26 @@ spec:
   lookupPolicy:
     local: true
   tags:
+  # N Version of the image
   - annotations:
       opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
-      opendatahub.io/notebook-python-dependencies: '[{"name":"TrustyAI","version":"0.2"}, {"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Elyra","version":"3.15"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"TrustyAI","version":"0.3"},{"name":"Boto3","version":"1.28"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.3"},{"name":"Scipy","version":"1.11"},{"name":"Elyra","version":"3.15"},{"name":"PyMongo","version":"4.5"},{"name":"pyodbc","version":"4.0"}, {"name":"Codeflare-SDK","version":"0.8.0"}, {"name":"sklearn-onnx","version":"1.15"}, {"name":"Psycopg2","version":"2.9"}, {"name":"MySQL Connector/Python","version":"8.0"}]'
```

Comment:
We need to keep only the x.y on the versions so the `Codeflare-SDK 0.8.0` should change to  `Codeflare-SDK 0.8`.
The `Psycopg2  2.9` don't exist instead there is the `psycopg 3.1`